scripts/bootstrap.bat

```diff
@@ -302,6 +311,19 @@ if "%install_python_deps%" == "1" (
     if %errorlevel% neq 0 goto script_failure
 )
 
+if "%virtspec%" == "1" (
+    echo.
+    echo %date% %time% ======================================================
+    echo %date% %time% * Appending PyUSB Virtual Spectrometer to PATH *
+    echo %date% %time% ======================================================
+    echo.
+    set PYTHONPATH=..\pyusb-virtSpec;%PYTHONPATH%
+    pip uninstall pyusb
+)
+if "%virtspec%" == "0" (
+    pip install pyusb==1.2.1
```

Comment:
If we're hardcoding pyusb version here, should that be in requirements.txt as well?

Reply:
Yes absolutely. I would like to keep that change out of this PR though. 

Because while I am certain that v1.2.1 is exactly the right pyusb for everyone using Enlighten everywhere-- I won't necessarily have that much confidence for every module we are pinning, and if we have problems down the line it would be nice to revert the 'version pinning' commit without rolling back the 'virtspec' one.